htsohm/material_files.py

```diff
@@ -290,31 +292,45 @@ def write_cif_file(file_name, lattice_constants, atom_sites):
         `$(raspa-dir)/structures/cif/(run_id)-(uuid).cif`
 
     """
+    htsohm_dir = os.path.dirname(os.path.dirname(htsohm.__file__))
+    run_dir = os.path.join(htsohm_dir, run_id)
+    material_dir = os.path.join(run_dir, 'pseudo_materials')
+    file_path = os.path.join(material_dir, "%s.yaml" % uuid)
```

Comment:
dumping structural data to yaml-files now factored into PseudoMaterial methods, need for this code partially eliminated